src/test/Components/CreateImageWizard/CreateImageWizard.test.js

```diff
@@ -1810,7 +1810,7 @@ describe('Keyboard accessibility', () => {
     clickNext();
 
     // Name
-    const nameInput = screen.getByRole('textbox', { name: /image name/i });
+    const nameInput = screen.getByRole('textbox', { name: /Image Name/i });
```

Comment:
We can leave the previous `{ name: /image name/i }` here as the `/i` means the check is case insensitive.